<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsExpressionStoreDialogBase</class>
 <widget class="QDialog" name="QgsExpressionStoreDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Store expression</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblLabel">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mLabel"/>
   </item>
   <item row="4" column="1">
    <widget class="QTextEdit" name="mHelpText"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblHelpText">
     <property name="text">
      <string>Help text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QgsCodeEditorExpression" name="mExpression" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblExpression">
     <property name="text">
      <string>Expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="mValidationError">
     <property name="text">
      <string>A stored expression with this name already exists!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCodeEditorExpression</class>
   <extends>QWidget</extends>
   <header>qgscodeeditorexpression.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
